versioned_docs/version-0.4.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -224,4 +225,5 @@ Editing KubeSlice serviceExportConfig...
 
 ## Uninstall KubeSlice
 
-To uninstall KubeSlice from your cloud clusters, see [Uninstall KubeSlice](/versioned_docs/version-0.4.0/kubeslice-cli/uninstalling-kubeslice.mdx).
\ No newline at end of file
+To uninstall KubeSlice from your cloud clusters, 
```

Comment:
To know how to uninstall KubeSlice from your cloud clusters,

Reply:
To know how is redundant here